games/h.yaml

```diff
@@ -344,6 +344,23 @@
   video:
     youtube: KyOkwXcEZgI
 
+- name: Hishigata
+  originals:
+  - Project Rhombus
+  type: clone
+  repo: 'https://github.com/LumpBloom7/hishigata'
+  development: sporadic
+  status: playable
+  content: open
+  langs:
+  - 'C#'
+  frameworks:
+  - osu!framework
```

Comment:
```suggestion
  - osu!
```

Reply:
also add this value to frameworks in dangerfile.js to silence the warning

Reply:
I believe we should match the official name instead: `osu!framework` -> https://github.com/ppy/osu-framework